lblod/ember-rdfa-editor-lblod-plugins
# Improve plugin translations

### Overview
This PR includes the following improvements:
- Addition of the `no-bare-strings` templating rule
- Internationalization of the roadsign-regulation plugin
- Internationalization of the import-snippet plugin
- Improvements in internalization of codelist and location edit components


### How to test/reproduce
Use the roadsign-regulation, import-snippet and variable plugins to check if no translations are missing or incorrect.

### Challenges/uncertainties
There are still several improvements that can be made to the internationalization of this addon:
- Check and remove usage of untranslated strings in javascript code
- Properly scope the translations of this addon so no interference happens with the consuming applications of this addon.
